src/components/LineCollection/index.tsx

```diff
@@ -93,7 +93,7 @@ const LineCollection: React.FC<Props> = ({
 
 export default withDisplayName('LineCollection', (props: Props) => (
   <ScalerContext.Consumer>
-    {({ domainsByItemId, subDomainsByItemId, series }: InternalProps) => (
+    {({ domainsByItemId, subDomainsByItemId, series }: any) => (
```

Comment:
Why to remove type?

Reply:
Because i need to rework the types on the context type